### ISSUE TITLE
Can't activate Gutenberg plugin with WP 5.3beta1

### ISSUE THREAD

**Issue by maddisondesigns** [WordPress/gutenberg#17565]
**Describe the bug**
I updated a site to WP5.3 (46288). When I tried to login I received an error indicating it was an issue with the Gutenberg plugin so I renamed the plugin to force WP to automatically deactivate it.

After doing this I was able to login to WP but when I went back to the Plugins screen to try to reactivate Gutenberg, I received the same error.

```
Plugin could not be activated because it triggered a fatal error.
Fatal error: Cannot declare class WP_Block_Styles_Registry, because the name is already in use in /Users/anthonyhortin/Documents/sites/dev.wordpress-test.com/wp-content/plugins/gutenberg/lib/class-wp-block-styles-registry.php on line 14
```

<img width="1922" alt="screenshot_208" src="https://user-images.githubusercontent.com/2078245/65564955-dcd96f00-df91-11e9-835f-6ba7b269a9fa.png">

**To reproduce**
1. Activate the Gutenberg plugin on a site
2. Update site to WP 5.3 beta 1
3. Try to login to the site again

**Expected behavior**
WP Shouldn't break

**Desktop (please complete the following information):**
- OS: macOS Mojave 10.14.6
- Browser: Firefox Quantum 69.0.1
- WP 5.3 Beta 1
- Gutenberg v6.4.0

**Comment by youknowriad** [comment]
Thanks for opening the issue, this is already fixed in Gutenberg v6.5 I believe.

**Comment by youknowriad** [comment]
or maybe it's in master and in that case it will be included in the next plugin release.
